ewah/hooks/shopify.py

```diff
@@ -255,16 +325,29 @@ def get_data(
                 # the request with new ids once the previous pagination is done
                 finished_pagination = False
 
-            response = requests.get(url, **req_kwargs)
             if is_first or not finished_pagination:
-                is_first = False
                 req_kwargs = kwargs_links
-            data = response.json().get(
-                object_metadata.get(
-                    "_name_in_request_data",
-                    shopify_object,
+
+            if parent_object == "fulfillment_orders":
+                response = requests.get(url, **req_kwargs, params=params)
+                if is_first:
+                    params = {}
+                    is_first = False
+
+            else:
+                # This is the main request for the connector used for most objects
+                response = requests.get(url, **req_kwargs)
+                is_first = False
+
+            # Special case: To avoid raising an exception we use other get method for data
+            if not shopify_object == "fulfillment_orders":
+                response.raise_for_status()
```

Comment:
(non-blocking) question: are we using this only for non fulfillment objects? why?